astropy/table/tests/test_row.py

```diff
@@ -340,7 +340,7 @@ def test_uint_indexing():
     """
     t = table.Table([[1., 2., 3.]], names='a')
     assert t['a'][1] == 2.
-    assert t['a'][np.int(1)] == 2.
+    assert t['a'][np.int_(1)] == 2.
```

Comment:
Why not `int` here like above?
```suggestion
    assert t['a'][int(1)] == 2.
```

Reply:
Here, I think the goal was really to use a guaranteed numpy int, which is what `np.int_(1)` gives; as written, the test just confirmed that a regular python int works...